.ci/magic-modules/create-pr.sh

```diff
@@ -110,6 +110,24 @@ if [ -n "$ANSIBLE_REPO_USER" ]; then
     fi
   fi
   popd
+
+  pwd
+
+  # If there is now a difference in the ansible_version_added files, those
+  # should be pushed back up to the user's MM branch to be reviewed.
+  if git diff --name-only HEAD^1 | grep "ansible_version_added.yaml"; then
+    git config --global user.email "magic-modules@google.com"
+    git config --global user.name "Modular Magician"
+
+    BRANCH=$(git config --get pullrequest.branch)
+    REPO=$(git config --get pullrequest.repo)
+
+    git add products/**/ansible_version_added.yaml
+    git commit -m "Ansible version_added changes"
+    
+    git remote add non-gcp-push-target "git@github.com:$REPO"
+    ssh-agent bash -c "ssh-add ~/github_private_key; git push -f non-gcp-push-target \"HEAD:$BRANCH\""
```

Comment:
I _think_ the `-f` is unnecessary - but I am not certain!  What do you think?  It scares me - imagine
1) I commit hash aaaaaaa
2) the magician runs on it.
3) I commit hash bbbbbbb _simultaneously with_
3) the magician clones in aaaaaaa, adds this version_added change (ccccccc), then
4) force-pushed ccccccc over my precious bbbbbbb change!

I also wonder if perhaps this pushes up a merge commit?  I think that HEAD here is the merge commit of `master` and the user's branch.

Reply:
I understand your worry about the -f flag (totally agree with it and think it should be removed).

I don't follow the rest.